src/components/layout/layout.js

```diff
@@ -73,6 +73,7 @@ const Layout = ({ children }) => {
                 </SidebarItems>
                 <SidebarItems to="/if-else/">If Else</SidebarItems>
                 <SidebarItems to="/for/">Loop For</SidebarItems>
+                <SidebarItems to="/spread-operator/">Spread Operator</SidebarItems>
                 <CloseButton
                   onClick={() => setIsVisible(!isVisible)}
                 ></CloseButton>
```

Comment:
@infinityover ,

Please merged with `master` branch

Thank you!

Reply:
@LeonardoCesca, sorry i didn't understood.
This line have a problem, or you will merge it into the master branch of the project?

Reply:
@infinityover,

please, merge the master of project into your branch and push a new commit

Reply:
@LeonardoCesca Done.
Thanks for the support.